internal/cmd/grants.go

```diff
@@ -60,19 +60,35 @@ func newGrantsListCmd(cli *CLI) *cobra.Command {
 
 			type row struct {
 				Identity string `header:"IDENTITY"`
-				Access   string `header:"ACCESS"`
+				Access   string `header:"ROLE"`
```

Comment:
might wanna hold off on updating this one until we have a design in place for what grants should be exposed like, just to keep things in sync